index.html

```diff
@@ -0,0 +1,46 @@
+<html>
+    <head>
+      <meta charset="UTF-8" />
+      <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>portfolio mobile</title>
+       <link rel="stylesheet" href="style.css">
+    </head>
+ <body>   
+      <header class="navbar navbar-expand-lg fixed-top navbarScroll">
+        
+          <a class="navbar__logo" href="#"> My Logo</a>
+          <div class="navbar__label_container">
+            <span class="__label__list_container"></span>
+            <span class="__label__list_container"></span>
+            <span class="__label__list_container"></span></div>
+
+        
+      </header>
+  
+      <section class="navigation">    
+            <h1 class="navigation__bar">I'm Alex Asciencio. Glad to see you!</h1>
+            <main>
+              <p class="me">
+                I'm a software developer! I can help you build a product , feature
+                or website Look through some of my work and experience! If you like
+                what you see and have a project you need coded, don't hesitate to
+                contact me.
+              </p>
+            </main>
```

Comment:
- [ ]  I think the `main tag` shouldn't be used to wrap the `paragraph`. Please follow this [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main) to read me about the `main` tag and how it should be used in sectioning the entire webpage :+1: